tests/integration/quteprocess.py

```diff
@@ -359,6 +359,18 @@ def open_path(self, path, *, new_tab=False, new_window=False, port=None,
         else:
             self.send_cmd(':open ' + url)
 
+    def open_url(self, url, *, new_tab=False, new_window=False):
```

Comment:
Can you adjust `open_path` to simply call `path_to_url` and then `open_url` instead of duplicating the code?